go.mod

```diff
@@ -10,5 +10,5 @@ require (
 	k8s.io/apimachinery v0.19.16
 	k8s.io/client-go v0.19.16
 	k8s.io/code-generator v0.19.16
-	knative.dev/pkg v0.0.0-20210331065221-952fdd90dbb0 // pin to branch release-0.22
```

Comment:
Yes, the updated knative.dev/pkg@release-0.22 fix the issue. I am equally why are we pinned to an older version. Maybe worth testing with the latest knative.dev/pkg@release-1.2. I will update my sandbox with the latest and try the samples.

Reply:
I just verified knative.dev/pkg@release-1.2 has breaking changes. So the latest patch from knative.dev/pkg@release-0.22 is sufficient for now.